dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -292,7 +292,7 @@ void CommandBarFlyoutCommandBar::UpdateVisualState(bool useTransitions)
             bool isConstrainedToRootBounds = true;
             auto controlBounds = TransformToVisual(nullptr).TransformBounds({ 0, 0, static_cast<float>(ActualWidth()), static_cast<float>(ActualHeight()) });
             
-#ifdef USE_INSIDER_SDK
+#ifdef USE_INTERNAL_SDK
```

Comment:
These APIs should be in the 18312 SDK, can you please include this under the "USE_INSIDER_SDK" ifdef?